Remotion/Web/Core/UI/Controls/TabbedMultiView.cs

```diff
@@ -288,8 +290,6 @@ private void OnTabViewRemove (TabView view)
         {
           if (MultiViewInternal.Controls.Count > 1)
             _newActiveTabAfterRemove = (TabView)MultiViewInternal.Controls[index - 1];
-          else // No Tabs left after this tab
-            _newActiveTabAfterRemove = _placeHolderTabView;
```

Comment:
this, you need to explain to me, please, because I can't verify the semantics in my head.